field_rrule/hooks.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import socket
+import inspect
+
+
+def post_load_hook():
+    """do some trickery to have demo data/model on runbot, but nowhere else"""
```

Comment:
I say we leave it like that for now, but a small voice in my head says that there is a better way 👼